sites/public/src/pages/sign-in.tsx

```diff
@@ -119,31 +121,76 @@ const SignIn = () => {
 
   useEffect(() => {
     if (
-      networkError?.error.response.data?.message ===
+      networkError?.error?.response?.data?.message ===
       EnumUserErrorExtraModelUserErrorMessages.accountNotConfirmed
     ) {
       setConfirmationStatusModal(true)
     }
   }, [networkError])
 
+  const SignUpBenefitsHeadingGroup = (props: { mobileView: boolean }) => {
+    const classNames = props.mobileView ? "py-6 px-4 -order-1" : "grow-0 shrink-1"
+    return (
+      <HeadingGroup
+        heading={t("account.signUpSaveTime.title")}
+        subheading={t("account.signUpSaveTime.subTitle")}
+        size="2xl"
+        className={classNames}
+      />
+    )
+  }
+
   return (
     <>
-      <FormsLayout>
-        <FormSignIn
-          onSubmit={(data) => void onSubmit(data)}
-          control={{ register, errors, handleSubmit, watch }}
-          networkStatus={{
-            content: networkStatusContent,
-            type: networkStatusType,
-            reset: () => {
-              reset()
-              resetNetworkError()
-              setConfirmationStatusMessage(undefined)
-            },
-          }}
-          showRegisterBtn={true}
-        />
-      </FormsLayout>
+      {process.env.showMandatedAccounts ? (
+        <FormsLayout className="sm:max-w-lg md:max-w-full">
+          <div className="flex flex-col md:flex-row md:ml-20 justify-center">
+            <div className="display md:hidden ">
+              <SignUpBenefitsHeadingGroup mobileView={true} />
+            </div>
+            <div className="md:max-w-lg w-full justify-center">
+              <FormSignIn
+                onSubmit={(data) => void onSubmit(data)}
+                control={{ register, errors, handleSubmit, watch }}
+                networkStatus={{
+                  content: networkStatusContent,
+                  type: networkStatusType,
+                  reset: () => {
+                    reset()
+                    resetNetworkError()
+                    setConfirmationStatusMessage(undefined)
+                  },
+                }}
+                showRegisterBtn={true}
+              />
+            </div>
+            <div className="hidden md:flex">
+              <div className="md:flex md:flex-col md:grow-0 md:shrink-1 md:p-8 md:max-w-lg md:w-full ">
```

Comment:
@cade-exygy At first glance, I couldn't see how the grow and shrink values were being utilized on the line above. Haven't used them before though so I could definitely be missing it! Just know that grow is defaulted to zero so maybe that can be removed? Trust your call here!

Reply:
Cleaned those styles up! Lmk if there is anything else!